spec/views/lectures/index.html.erb_spec.rb

```diff
@@ -2,7 +2,7 @@
 
 RSpec.describe "lectures/index", type: :view do
   before(:each) do
-    assign(:lectures, [
+    @lectures = assign(:lectures, [
```

Comment:
Could/Should we use here a factory bot ?
- [ ] Check if usage of factorybot would be better, if yes implement

Reply:
thanks, I am using the factory bot now